dune/grid/common/ZoltanPartition.cpp

```diff
@@ -53,11 +54,24 @@ std::vector<int> zoltanGraphPartitionGridOnRoot(const CpGrid& cpgrid,
     Zoltan_Set_Param(zz, "RETURN_LISTS", "ALL");
     Zoltan_Set_Param(zz, "DEBUG_LEVEL", "3");
     Zoltan_Set_Param(zz, "CHECK_GRAPH", "2");
+    Zoltan_Set_Param(zz,"EDGE_WEIGHT_DIM","0");
+    Zoltan_Set_Param(zz, "OBJ_WEIGHT_DIM", "0");
     Zoltan_Set_Param(zz, "PHG_EDGE_SIZE_THRESHOLD", ".35");  /* 0-remove all, 1-remove none */
 
     bool pretendEmptyGrid = cc.rank()!=root;
+    std::shared_ptr<CombinedGridWellGraph> grid_and_wells;
 
-    Dune::cpgrid::setCpGridZoltanGraphFunctions(zz, cpgrid, pretendEmptyGrid);
+    if( eclipseState )
+    {
+        Zoltan_Set_Param(zz,"EDGE_WEIGHT_DIM","1");
```

Comment:
>  I have seen that using the weighting function log(1/T) with special treatment of connections for which T=0 tends to give reasonable partitions

@bska Could you comment on why log(1/T)? If I look at `DerivedGeology::trans_` then the entries seem to be in the range 10^-14 to 10^-12 for Norne.  The formulation would mean that lower transmissibilities produce higher weights. But in my humble understanding we get higher flow over faces with high transmissibility. What am I missing?